examples/chol.dx

```diff
@@ -1,39 +1,12 @@
-' # Cholesky Factorization
-https://en.wikipedia.org/wiki/Cholesky_decomposition
+' ### PSD solver based on Cholesky decomposition
```

Comment:
I guess we might want to rename this file to `psd.dx` now

Reply:
Right fixed